### PR TITLE
feat(docker): use shorter prefix

### PR DESCRIPTION
**Pull request by serge1peshcoff**
see https://github.com/AEGEE/oms-core-js/pull/28

### CODE REVIEW

File: docker/docker-compose.dev.yml [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+    mailer:
         build:
-            context: ./${PATH_OMS_MAILER}/../
-            dockerfile: ./docker/oms-mailer/Dockerfile
+            context: ./${PATH_MAILER}/../
+            dockerfile: ./docker/mailer/Dockerfile
             args: 
                 MIX_ENV: development
-        image: aegee/oms-mailer:dev
+        image: aegeemailer:dev
```
Comment (WikiRik) [inline docker/docker-compose.dev.yml:10]: aegee/mailer:dev?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/AEGEE/oms-mailer.svg?branch=master)](https://travis-ci.org/AEGEE/oms-mailer)
+[![Build Status](https://travis-ci.org/AEGEE/mailer.svg?branch=master)](https://travis-ci.org/AEGEE/mailer)
```
Comment (WikiRik) [inline README.md:1]: Maybe update this to Circle CI as well?

File: docker/docker-compose.dev.yml [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
             - /usr/src/myapp/deps
 
-    oms-mail-transfer-agent:
+    mail-transfer-agent:
         build:
-            context: ./${PATH_OMS_MAILER}/..
+            context: ./${PATH_MAILER}/..
             dockerfile: docker/mail_transfer_agent/Dockerfile
-        image: aegee/oms-mail-transfer-agent:dev
+        image: aegeemail-transfer-agent:dev
```
Comment (WikiRik) [inline docker/docker-compose.dev.yml:18]: aegee/mail-transfer-agent:dev?

File: docker/docker-compose.yml [hunk 4]
```diff
@@ -19,17 +19,20 @@ services:
...
         labels:
-            - "traefik.backend=oms-mailer"
+            - "traefik.backend=mailer"
             - "traefik.port=4000"
-            - "traefik.frontend.rule=Path:/services/oms-mailer/api/healthcheck;PathPrefixStrip:/services/oms-mailer/api"
+            - "traefik.old.frontend.rule=Path:/services/oms-mailer/api/healthcheck;PathPrefixStrip:/services/oms-mailer/api"
+            - "traefik.new.frontend.rule=Path:/api/mailer/healthcheck;PathPrefixStrip:/api/mailer"
+            - "traefik.old.frontend.priority=110"
+            - "traefik.new.frontend.priority=110"
             - "traefik.frontend.priority=110"
```
Comment (WikiRik) [inline docker/docker-compose.yml:28]: Will Traefik use this priority? Since the priority for both the old and the new have also been specified

Comment (serge1peshcoff) [reply docker/docker-compose.yml:28]: Here, the priority doesn't matter as these endpoints have different prefixes and are not overlapping each other. Main thing is that these should override the frontend priorities, so requests like this won't be sent to frontend.

File: README.md (hunk 2 above)
Comment (serge1peshcoff) [reply README.md:1]: Nor sure if CircleCI provides badges, will check later.

File: docker/docker-compose.dev.yml (hunk 3 above)
Comment (serge1peshcoff) [reply docker/docker-compose.dev.yml]: thanks, didn't notice. fixed now

File: README.md (hunk 2 above)
Comment (serge1peshcoff) [reply README.md]: yeah, it does, updated it. I've changed it to mailer as I'll rename the repo before merging.

File: docker/docker-compose.dev.yml (hunk 1 above)
Comment (serge1peshcoff) [reply docker/docker-compose.dev.yml]: thanks again! should be fixed now.

**Pull request merged**